### DISCUSSION TITLE
Pages API deprecated?

### DISCUSSION THREAD

**Discussion by kennym** [nandorojo/solito#90] [Q&A] [answered]
I went through the showtime repo and found lack of use of the NextJS Pages API apparent. Is there any reason why that is?

Instead I saw the use of `useSWR` which is nice. Is that a replacement for the Pages API?

**Comment by nandorojo** [comment, marked as answer]
hmm not sure i know what you mean. see their `apps/next` folder for pages. SWR is just used for fetching data.
